pkg/virt-operator/resource/apply/instancetype_test.go

```diff
@@ -57,31 +68,30 @@ var _ = Describe("Apply Instancetypes", func() {
 				},
 			}
 
+			reconciler.targetStrategy = &install.Strategy{}
+			reconciler.targetStrategy.SetInstancetypes([]*instancetypev1beta1.VirtualMachineClusterInstancetype{instancetype})
+
 			imageTag, imageRegistry, id := getTargetVersionRegistryID(reconciler.kv)
 			injectOperatorMetadata(reconciler.kv, &instancetype.ObjectMeta, imageTag, imageRegistry, id, true)
 		})
 
 		It("should create instancetype", func() {
-			getCalled := expectGetReturnNotFound(fakeClient, apiinstancetype.ClusterPluralResourceName)
 			createCalled := expectCreate(fakeClient, apiinstancetype.ClusterPluralResourceName, instancetype)
 			Expect(reconciler.createOrUpdateInstancetype(instancetype)).To(Succeed())
-			Expect(*getCalled).To(BeTrue())
 			Expect(*createCalled).To(BeTrue())
 		})
 
 		It("should not update instancetypes on sync when they are equal", func() {
-			getCalled := expectGet(fakeClient, apiinstancetype.ClusterPluralResourceName, instancetype)
+			reconciler.stores.ClusterInstancetype.Add(instancetype)
```

Comment:
Can we assert create/update was not called?

Reply:
https://github.com/kubevirt/kubevirt/blob/681ccd0543235f87975480b05e71b057339790f5/pkg/virt-operator/resource/apply/instancetype_test.go#L34-L38